test/run_tests.py

```
@@ -0,0 +1,110 @@
+    print("RELINKER TEST SUITE")
+    print("=" * 70)
+    print()
+    
+    # Add unit tests
+    print("Loading unit tests...")
+    try:
+        import test_configuration
+        configuration_tests = loader.loadTestsFromModule(test_configuration)
+        suite.addTests(configuration_tests)
+        print(f"  ✓ Loaded {configuration_tests.countTestCases()} tests from test_configuration")
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)